packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -323,11 +340,21 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 					miniflare
 				);
 
-				const miniflareDevOptions = await getDevMiniflareOptions(
+				let containerBuildId: string | undefined;
+				const workerConfig = getEntryWorkerConfig(resolvedPluginConfig);
+				if (
+					workerConfig?.containers?.length &&
+					workerConfig.dev.enable_containers
+				) {
```

Comment:
```suggestion
        const hasDevContainers = entryWorkerConfig?.containers?.length && entryWorkerConfig.dev.enable_containers
				if (hasDevContainers) {
```

Let's make this a variable as it's also used later on.